core/domain/user_services_test.py

```diff
@@ -1899,6 +1934,30 @@ def test_set_user_has_viewed_lesson_info_modal_once(self):
         self.assertTrue(
             user_settings_model.has_viewed_lesson_info_modal_once)
 
+    def test_log_username_change(self):
+        def mock_get_current_time_in_millisec():
+            return 1.00
+
+        with self.swap(
+            utils, 'get_current_time_in_millisecs',
+            mock_get_current_time_in_millisec):
+            committer_id = 'someUser'
+            audit = audit_models.UsernameChangeAuditModel()
```

Comment:
This is a bit weird. Why are you making an instance of the model class? Normally we just call UsernameChangeAuditModel.get().

Reply:
I changed it to just using UsernameChangeAuditModel.get() instead!